robot/game.py

```diff
@@ -1,10 +1,37 @@
+import logging
+import time
 from enum import Enum
+from threading import Thread
 from typing import NewType
 
+import _thread
 from robot.board import Board
 
 Zone = NewType('Zone', int)
 
+LOGGER = logging.getLogger(__name__)
+
+
+def kill_after_delay(timeout_seconds, exit_message):
+    """
+    Interrupts main process after the given delay.
+    """
+
+    end = time.time() + timeout_seconds
```

Comment:
I don't think that `end` is a particularly great variable name here.

Perhaps `end_time`?